product_multiple_barcodes/views/product_template_views.xml

```diff
@@ -68,4 +68,29 @@
         </field>
     </record>
 
+    <record id="product_product_view_form_multiply_barcode" model="ir.ui.view">
+        <field name="name">product.product.view.form.multiply.barcode</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_normal_form_view"/>
+        <field name="arch" type="xml">
+                <header position="inside">
+                    <button string="Update Barcode" type="action"
+                        name="%(product_multiple_barcodes.action_multiply_barcode_wizard)d"/>
+                </header>
+        </field>
+    </record>
+
+    <record id="product_template_view_form_multiply_barcode" model="ir.ui.view">
+        <field name="name">product.template.view.form.multiply.barcode</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="arch" type="xml">
+                <header position="inside">
+                    <button string="Update Barcode" type="action"
+                        name="%(product_multiple_barcodes.action_multiply_barcode_wizard)d"
+                        attrs="{'invisible': [('product_variant_count', '!=', 1)]}"/>
```

Comment:
Maybe we should not show this button in case product is not yet created? Like we do not show barcode_ids field untill product is created.